packages/dbml-core/src/parse/dbml/src/lib/parser/nodes.ts

```diff
@@ -0,0 +1,575 @@
+import { SyntaxToken } from '../lexer/tokens';
+
+export interface SyntaxNode {
+  kind: SyntaxNodeKind;
+  start: number;
```

Comment:
Does this start count from node's leading trivia (comments, etc.) or from the node's start itself?

Reply:
It counts from the content start, not including the trivia. It defines the start of the yellow highlight in the playground. Should I name it more explicitly?